src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/SelectVariants.java

```diff
@@ -472,6 +478,16 @@ protected GenomicsDBOptions getGenomicsDBOptions() {
     public void onTraversalStart() {
         final Map<String, VCFHeader> vcfHeaders = Collections.singletonMap(getDrivingVariantsFeatureInput().getName(), getHeaderForVariants());
 
+        List<String> genotypeField = getHeaderForVariants().getGenotypeSamples();
+        if(!ParsingUtils.isSorted(genotypeField)){
+            logger.warn("Detected unsorted genotype fields on input. This could result in very slow traversal for large files.");
```

Comment:
#7732 give how dangerous the risk of 10x slower runs for very large VCF files I would recommend making this warning much louder and hard to miss. I would recommend making the warning behave like the warning in `HaplotypeCaller.onTraversalStart()` so it is much more visible. I would also change the text here "This could result in very slow traversal for large files" -> "SelectVariants will sort the genotypes on output which will result in very slow traversal on large inputs" or something along those lines.

Reply:
Ideally this warning could be contextual and only happen if we would otherwise not unpack the genotypes, but that might be difficult to check for.

Reply:
Honestly tying the warning to the filesize/the number of genotypes in the first place might also be worth it.

Reply:
yeah, make the size of the warning font be dependent on the number of genotypes.

Reply:
![image](https://user-images.githubusercontent.com/4700332/172918949-37ffbb86-5c7a-4883-b633-2e73d07ddb41.png)